server/datastore/mysql/software_installers.go

```diff
@@ -3101,17 +3101,18 @@ func (ds *Datastore) getIncludedHostUUIDMapForSoftware(ctx context.Context, tx s
 FROM
 		hosts h
 		JOIN android_devices ad ON ad.enterprise_specific_id = h.uuid
+		JOIN vpp_apps_teams vat ON vat.team_id <=> h.team_id AND vat.id = ?
```

Comment:
Was a bit surprised by `<=>`, what does that do?

Reply:
@mna also a TIL for me, it's nifty!